examples/prompting/Basic_Reasoning.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
<p>Can we provide any narrative text to go with this?</p><p> </p><p>Not sure if we need to or not - honestly this is pretty clear as-is - but even a short explanation for why the template &amp; problem are separated (e.g. "First you set the template, this is reusable across different problems...")</p><p> </p><p>And this part should go in the system instruction, with the problem passed directly to generate_content.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>